### PR TITLE
Nuimo app role

### PR DESCRIPTION
**Pull request by grunskis**
Automate setup installation of Nuimo app.

To be used together with https://github.com/getsenic/senic-hub-nuimo-app/pull/1

### CODE REVIEW

File: deployment/roles/hub-nuimo-app/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  copy:
+    dest: "{{deploy_location}}/requirements.txt"
+    owner: "{{deploy_user}}"
+    group: "{{run_user}}"
+    mode: "0644"
+    content: |
+      pip==9.0.1
+      nuimo==0.1.0
+      websocket-client==0.40.0
+      https://github.com/grunskis/senic-hub-nuimo-app/archive/ftw.zip#egg=nuimo-app==0.0.1
```
Comment (tomster) [inline deployment/roles/hub-nuimo-app/tasks/main.yml]: we should make a release on our devpi server and install from there. this makes sure, that the package is correct and can 'survive' the packaging process

File: deployment/roles/hub-nuimo-app/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+- name: install application
+  pip:
+    virtualenv: "{{deploy_location}}/venv/"
+    requirements: "{{deploy_location}}/requirements.txt"
+    extra_args: "--pre --upgrade -i {{devpi_index_url}}"
+  notify: restart nuimo app
+
+- name: enable virtualenv to access the system packages
```
Comment (tomster) [inline deployment/roles/hub-nuimo-app/tasks/main.yml]: why not simply use ``--with-site-packages`` when creating the virtualenv?

File: deployment/roles/hub-nuimo-app/tasks/main.yml (hunk 1 above)
Comment (grunskis) [inline deployment/roles/hub-nuimo-app/tasks/main.yml]: Ok. Once it lands in master I'll create a tag and a release. Actually I could use your help with releasing it...

File: deployment/roles/hub-nuimo-app/tasks/main.yml (hunk 2 above)
Comment (grunskis) [inline deployment/roles/hub-nuimo-app/tasks/main.yml]: That was my initial approach. But then I noticed that this prevents `pip` to be installed within the `virtualenv`. So in the end I found that this is the easiest approach to achieve same thing with `venv` what was the usual behavior with `virtualenv`

File: deployment/roles/hub-nuimo-app/tasks/main.yml (hunk 1 above)
Comment (tomster) [inline deployment/roles/hub-nuimo-app/tasks/main.yml]: yes, i would like to use the automatic versioning scheme, but we first need to review and merge https://github.com/getsenic/senic-hub-nuimo-app/pull/1 anyway, so we can do that there and then adjust the requirement here.

IOW not a showstopper

**Pull request merged**